app/controllers/orders_controller.rb

```diff
@@ -1,4 +1,4 @@
-class OrdersController < ApplicationController
+class Or<% <%  %> %>dersController < ApplicationController
```

Comment:
unexpected token tGT
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)<br>unterminated string meets end of file
(Using Ruby 2.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)